lib/improver/generate_ancillaries/generate_ancillary.py

```diff
@@ -227,42 +228,55 @@ def gen_orography_masks(
         thresholds = Unit(units).convert(
             np.array(thresholds), standard_orography.units)
         coords = standard_orography.coords()
-        if key == 'land':  # regular topographical bands above land
-            lower_threshold, upper_threshold = thresholds
-            orog_band = np.ma.masked_where(
-                np.ma.logical_and(
-                    (standard_orography.data > lower_threshold),
-                    (standard_orography.data <= upper_threshold)),
-                standard_orography.data).mask.astype(int)
+
+        lower_threshold, upper_threshold = thresholds
+        orog_band = np.ma.masked_where(
+            np.ma.logical_and(
+                (standard_orography.data > lower_threshold),
+                (standard_orography.data <= upper_threshold)),
+            standard_orography.data).mask.astype(int)
+
+        if key == 'land':  # topographical bands above land only
```

Comment:
Could be worth adding a list of permitted keys to this function's docstring?

Reply:
✅